WordPress/gutenberg
# Position: Refactor "Position" controls panel to use `ToolsPanel` instead of `PanelBody`

Fixes: #67950 

## What?
This PR introduces the usage of `ToolsPanel` instead of `PanelBody` inside the `Position` block support UI.

## Why?
This PR is a part of #67813 which standardizes the usage of `ToolsPanel` inside `Block Support UI`.

## How?
`PanelBody` is replaced with the corresponding `ToolsPanel` component.

## Testing Instructions
1. Insert a Group Block.
2. Notice the presence of `ToolsPanel` component.
3. Try to reset the updated values to test the reset functions.

## Screencast

https://github.com/user-attachments/assets/6b11a241-e8f6-4c72-88a2-66476e6e2c68
